sentinel/src/schedule/replications.js

```diff
@@ -0,0 +1,70 @@
+const config = require('../config'),
+      later = require('later'),
+      db = require('../db');
+
+// set later to use local time
+later.date.localTime();
+
+function isConfigValid(config) {
+  // Failing parsing will throw an Error
+  try {
+    return Boolean(
+      config.from &&
+      config.to &&
+      (config.text_expression  && later.parse.text(config.text_expression) || 
+       config.cron && later.parse.cron(config.cron))
+    );
+  } catch(e) {
+    return false;
+  }
+}
+
+function getSchedule(config) {
+  // fetch a schedule based on the configuration, parsing it as a "cron"
+  // or "text" statement see:
+  // http://bunkat.github.io/later/parsers.html
+  if (!config) {
+    return;
+  }
+  if (config.text_expression) {
+    // text expression takes precedence over cron
+    return later.parse.text(config.text_expression);
+  }
+  if (config.cron) {
+    return later.parse.cron(config.cron);
+  }
+}
+
+function replicate(fromDbs, toDb) {
+	const targetDb = db.get(toDb);
+  return fromDbs.reduce((p, fromDb) => {
+    return p.then(() => db.get(fromDb).replicate.to(targetDb));
```

Comment:
Awesome! One more small thing - it'd be good to log what's going on. I think in this `reduce` maybe add something like...

```js
logger.info(`Replicating docs from "${fromDb}" to "${toDb}"`);
```

This will help us debug if something goes wrong!

Reply:
Also, I just realised we probably don't want to replicate all docs because the userdb contains read docs which aren't useful as far as I can tell. Also, because the ID is based on the uuid of the doc that has been read, if two users view the same doc they will conflict. There's also the _design doc to worry about...

So I think we need to filter replication so we only replicate feedback and telemetry docs - we can add more later if we want. Something like this...

```js
return p.then(() => db.get(fromDb).replicate.to(targetDb, {
  filter: doc => doc._id.startsWith('telemetry-') || doc._id.startsWith('feedback-')
}));
```

That line is starting to be hard to read so consider pulling the function out somewhere else so it can be named and commented on etc...